apps/prairielearn/src/pages/partials/submission.ejs

```diff
@@ -75,7 +75,15 @@
           <% } %>
         </span>
       </div>
-      <small>Submitted at <%- submission.formatted_date %></small>
+      <span class="small">
+        <% if (!submission.user_uid) { %>
+          Submitted at <%= submission.formatted_date %> 
+        <% } else if (authz_data.authn_user.uid === submission.user_uid) { %>
```

Comment:
TODO before merging: `authz_data` is not available when we're rendering this partial outside the context of a full page load (that is, when loading old submissions on the fly). @eliotwrobson how important do you think it is to be "clever" and show "You" instead of the user's UID? We could save some complexity if we're willing to give that up.

Reply:
I think just showing the uid instead of You is probably the way to go, I don't think the extra cleverness is really doing much in this case.

Reply:
Removed in 6ebc5c6.